### DISCUSSION TITLE
Get currency symbol along with excel cell value

### DISCUSSION THREAD

**Discussion by iamsethi** [ozlerhakan/poiji#284] [General]
Currently working on an excel which is having a currency symbol along with the cell value as shown in snapshot 
Wondering if there is any way to get currency symbol along with the cell value?

<img width="264" alt="image" src="https://github.com/ozlerhakan/poiji/assets/12969857/0544ed1a-74b7-42ce-8abd-c7bf3a07cc68">

**Comment by ozlerhakan** [comment]
Hi @iamsethi ,

I have never experienced this particular case before. What do you get with the current logic? Did you create your own Casting class to debug these values?
